README_new.md

```diff
@@ -0,0 +1,617 @@
+# Orion Context Broker
+
+This is the code repository for the Orion Context Broker, the reference implementation of the Publish/Subscribe Context Broker GE.
+
+This project is part of FIWARE.
+
+* [FIWARE Site](http://www.fiware.org)
+* [FIWARE Catalogue for Orion](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker)
+
+Any feedback on this documentation is highly welcome, including bugs, typos
+or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-orion/issues/new) to provide feedback.
+
+For documentation previous to Orion 0.23.0 please check the old manuals at FIWARE public wiki:
+
+* [Orion Context Broker - Installation and Administration Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide)
+* [Orion Context Broker - User and Programmers Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide)
+
+## GEi overall description
+
+Orion is a C++ implementation of the NGSI9/10 REST API binding developed as a part of the FIWARE platform.
+
+The Orion Context Broker is an NGSI9/10 server implementation to manage context information and context information availability. Using the Orion Context Broker, you are able to register context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. an interval of time has passed or the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this document.
+
+If this is your first contact with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/md/quick_start_guide.md).
```

Comment:
On 25/06/15 16:56, Jose M. Cantera wrote:

In README_new.md<https://github.com/telefonicaid/fiware-orion/pull/1024#discussion_r33263699>:

> +
> +Any feedback on this documentation is highly welcome, including bugs, typos
> +or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-orion/issues/new) to provide feedback.
> +
> +For documentation previous to Orion 0.23.0 please check the old manuals at FIWARE public wiki:
> +
> +* [Orion Context Broker - Installation and Administration Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide)
> +* [Orion Context Broker - User and Programmers Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide)
> +
> +## GEi overall description
> +
> +Orion is a C++ implementation of the NGSI9/10 REST API binding developed as a part of the FIWARE platform.
> +
> +The Orion Context Broker is an NGSI9/10 server implementation to manage context information and context information availability. Using the Orion Context Broker, you are able to register context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. an interval of time has passed or the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this document.
> +
> +If this is your first contact with Orion Context Broker, it is highly recommended to have a look to the brief [Quick Start guide](doc/md/quick_start_guide.md).


your first approach to ...

—
Reply to this email directly or view it on GitHub<https://github.com/telefonicaid/fiware-orion/pull/1024/files#r33263699>.


Better with "your first contact with".

"Approach" as a noun means 'a way to do things', and this is not what we want here ...



________________________________

Este mensaje y sus adjuntos se dirigen exclusivamente a su destinatario, puede contener información privilegiada o confidencial y es para uso exclusivo de la persona o entidad de destino. Si no es usted. el destinatario indicado, queda notificado de que la lectura, utilización, divulgación y/o copia sin autorización puede estar prohibida en virtud de la legislación vigente. Si ha recibido este mensaje por error, le rogamos que nos lo comunique inmediatamente por esta misma vía y proceda a su destrucción.

The information contained in this transmission is privileged and confidential information intended only for the use of the individual or entity named above. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution or copying of this communication is strictly prohibited. If you have received this transmission in error, do not read it. Please immediately reply to the sender that you have received this communication in error and then delete it.

Esta mensagem e seus anexos se dirigem exclusivamente ao seu destinatário, pode conter informação privilegiada ou confidencial e é para uso exclusivo da pessoa ou entidade de destino. Se não é vossa senhoria o destinatário indicado, fica notificado de que a leitura, utilização, divulgação e/ou cópia sem autorização pode estar proibida em virtude da legislação vigente. Se recebeu esta mensagem por erro, rogamos-lhe que nos o comunique imediatamente por esta mesma via e proceda a sua destruição